lib/commands/new.js

```diff
@@ -102,6 +104,19 @@ module.exports = async (cwd, dir, ctx) => {
     await Helpers.exec('git add package.json README.md', { cwd: dir });
 };
 
+// TODO Give credit
```

Comment:
What is actionable for this TODO?  Let's resolve this!

Reply:
🤦 lol, shit. I'd meant to mention this, even peeked that project's license, then promptly spaced. cool! Solution below looks chill!